content/03.text-mining.md

```diff
@@ -21,34 +21,67 @@
-	   1. 10.1016/j.artmed.2004.07.016 
-	   2. 10.1186/1471-2105-8-50 
-	   3. Learning language in logic - genic interaction extraction challenge
-	   4. 10.1093/bioinformatics/btl616 
-	   5. http://helix-web.stanford.edu/psb02/ding.pdf
-	2. DaG - 3 datasets
-	   1. 10.1016/j.jbi.2012.04.004 
-	   2. 10.1186/s12859-015-0472-9
-	   3. 10.1186/1471-2105-14-323 
-	   4. 10.1186/1471-2105-13-161
-	3. CiD 
-	  1. 10.1093/database/baw068 
-	4. CbG 
-	  1. Biocreative VI track 5 - raw citation
-	5. more if exists talk about deep learning methods
-2. Mention the use of Support Vector Machines and other non deep learning classifiers
-   1. Will have to mention that field has moved to deep learning.
-   2. 10.1186/s13326-017-0168-3
-   3. 10.1371/journal.pcbi.1004630
-3. Mention deep learning methods
-   1. 1901.06103v1
-   2. 10.1016/j.knosys.2018.11.020
-   3. 10.1177/0165551516673485
-   4. 1706.01556v2
-   5. ^^ A few papers here but a lot more will be put into place 
-   6. Mention caveat which is the need for large annotated datasets
-   7. Mention a direction the field is moving to which is weak supervision and more that info that will come in time.
+#### Supervised Relationship Extraction
+
+Supervised extraction uses labeled data to learn patterns and map example input to desired output.
+Most of these approaches have flourished due to publically available datasets (Table {@tbl:supervised-text-datasets}).
+These datasets were constructed by curators for shared open tasks [@raw:biocreative/chemprot; @doi:10.1093/database/baw068] or as a means to provide the scientific community with a gold standard.
+Approaches that use these available datasets range from using support vector machines (SVMs) with custom kernels to deep learning with algorithms that can construct their own features.
+In the rest of this section we discuss approaches that use supervised methods to detect relationship asserting sentences.
+
+Extracting relationships in a supervised setting requires the capacity to map textual input onto a high dimensional space.
+Support vector machines are a type of classifier that can accomplish this mapping problem by utilizing a mapping function called a kernel [@doi:10.1186/s13326-017-0168-3; @doi:10.1371/journal.pcbi.1004630].
+These kernels take information such as a sentence's dependency tree [@doi:10.1093/database/bay108; @doi:10.1093/bioinformatics/btw503], part of speech tags [@doi:10.1186/s13326-017-0168-3] or even word counts [@doi:10.1371/journal.pcbi.1004630] and directly maps them onto a dense feature space.
+Within this space a hyperplane can be constructed to seperate sentences into the positive class (mentions a relationship) or negative class (does not mention a relationship). 
+Constructing these kernels necessitates the use of expert knowledge to be successful.
+Similar to regular manual curation, relying on experts to manually construct functions result in highly precise performance, but suffer overall in terms of recall.
```

Comment:
I don't quite understand this - do you mean that the kernels are manually constructed/selected? And does it really give performance similar to manual curation? I'd expect them to be pretty different.